### ISSUE TITLE
Parsing wifi settings may fail

### ISSUE THREAD

**Issue by martignoni** [martignoni/moodle-tool_moodlebox#19]
Using PHP function `parse_ini_file()` to parse file `/etc/hostapd/hostapd.conf` is not reliable.

**Comment by martignoni** [comment]
Find another way to get wifi settings to replace `parse_ini_file()`.

**Comment by martignoni** [comment]
Possible solution:

```php
function my_parse_ini_file($file,$mode=false,$scanner_mode=INI_SCANNER_NORMAL) {
  return parse_ini_string(preg_replace('/^#.*\\n/m', "", @file_get_contents($file)),$mode,$scanner_mode);
}
```

**Comment by martignoni** [comment]
Fixed by 04ada30, d93b527 and 0f5fae1.
